test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test

```diff
@@ -134,7 +134,7 @@ echo
 =======================================================
 HTTP/1.1 201 Created
 Content-Length: 0
-Location: /v2/entities/E1
+Location: /v2/entities/E1?type=T
```

Comment:
The "Location: ..." examples in the NGSIv2 .apib should be also adjusted in a similar way.